lib/improver/cli/apply_emos_coefficients.py

```diff
@@ -293,22 +328,37 @@ def process(current_forecast, coeffs, landsea_mask, num_realizations=None,
     calibrated_predictor, calibrated_variance = ac.process(
         current_forecast, coeffs, landsea_mask=landsea_mask)
 
+    # Handle numbers or inf as shape parameters.
+    if shape_parameters:
+        accepted_params = {"inf": np.inf, "-inf": -np.inf}
+        for param_index, param in enumerate(shape_parameters):
+            if param in accepted_params.keys():
+                shape_parameters[param_index] = accepted_params[param]
+            else:
+                shape_parameters[param_index] = float(param)
```

Comment:
Why does the user not simply specify np.inf, with appropriate instructions to do so in the doc-string?

Reply:
I've removed this logic, as suggested, so the user just specifies "inf" on the command line, which gets interpreted as a float when read into the `shape_parameters` argument.